src/datasource/dsBridge.js

```diff
@@ -0,0 +1,3 @@
+import { NativeModules } from 'react-native';
+
+module.exports = NativeModules.DSBridge;
```

Comment:
This should match the name in name in your `getName()` method here https://github.com/applicaster/applicaster-android-sdk/blob/914c5420761cd613b71bea958cf8c45fe9e318df/src/main/java/com/applicaster/reactnative/ZappPipes.java#L28
so it should be `NativeModules.ZappPipes`

also, try to avoid mixing different types of module statements - since you're using `import` you can replace `module.exports = NativeModules.DSBridge` by
```
const { ZappPipes } = NativeModules;
export default ZappPipes; 
```

Reply:
you right, i forgot to push it after i changed the name in the android sdk.